src/navigators/RootStack.tsx

```diff
@@ -19,7 +19,7 @@ export type NotLoggedInRootParamList = {
 export function RootStack() {
   const { isLoggedIn, loading } = useAuth();
 
-  if (loading) {
+  if (!isLoggedIn && loading) {
```

Comment:
This might evolve, but for now it provides the experience we're used to.  Without this change, any token refresh would bring the user back to the `Waiting for authorization` full-screen (below) and then home screen afterwards (if the refresh succeeded).  That might be desired functionality for some apps, so I could see this becoming configurable someday.  The approach in this PR means components/screens may still be on screen after errors (even 401s) and need to be prepared for error display and/or refresh controls to accommodate that scenario.  Again this is what we're used to so that can be the default initially.